### PR TITLE
Fix-published-flag

### PR DESCRIPTION
**Pull request by gyfchong**
## Why
Boolean logic hard

## What
Fix boolean logic

### CODE REVIEW

File: .github/workflows/changeset.yaml [hunk 1]
```diff
@@ -31,22 +32,15 @@ jobs:
...
-          slack-message: "Packages have been published"
-        env:
-          SLACK_BOT_TOKEN: ${{ secrets.SLACK_BOT_TOKEN }}
-      - name: Publish failed
-        if: ${{ env.PUBLISHED == 'false' }}
-        uses: slackapi/slack-github-action@v1.24.0
-        with:
-          channel-id: "C02NUQ27G56"
-          slack-message: "Publish failed"
+          slack-message: ${{ env.PUBLISHED == 'true' && 'Packages have been published' || 'Publishing failed' }}
```
Comment (HeartSquared) [inline .github/workflows/changeset.yaml:44]: This is interesting... Is this the equivalent of a ternary? 😂

Comment (gyfchong) [reply .github/workflows/changeset.yaml:44]: Yes! Haha it's super weird

File: .github/workflows/changeset.yaml [hunk 2]
```diff
@@ -30,23 +29,17 @@ jobs:
           publish: yarn ci:publish
   notify-slack:
     runs-on: ubuntu-latest
-    needs: version
+    needs:
+      version
+      # We only trigger this flow if the publish is happening, inferred by the presence of a changeset.
```
Comment (HeartSquared) [inline .github/workflows/changeset.yaml:34]:
```suggestion
      # We only trigger this flow if the publish is happening, inferred by not having a changeset.
```

File: .changeset/stupid-pumpkins-begin.md [hunk 3]
```diff
@@ -0,0 +1,5 @@
+---
+"@kaizen/components": patch
+---
+
+Tester publish
```
Comment (HeartSquared) [inline .changeset/stupid-pumpkins-begin.md:5]: Do you still want to keep this?

Comment (gyfchong) [reply .changeset/stupid-pumpkins-begin.md:5]: Yeah, still want to see if the ternary works

**Review by HeartSquared — APPROVED** [review]

**Pull request merged**